amoro-ams/amoro-ams-server/src/main/java/org/apache/amoro/server/table/TableOptimizingMetrics.java

```diff
@@ -163,6 +165,37 @@ public class TableOptimizingMetrics {
           .withTags("catalog", "database", "table")
           .build();
 
+  public static final MetricDefine TABLE_LAST_MINOR_OPTIMIZING_INTERVAL =
+      defineGauge("table_last_minor_optimizing_duration_mills")
+          .withDescription("Duration in milliseconds after last minor optimizing")
+          .withTags("catalog", "database", "table")
+          .build();
+
+  public static final MetricDefine TABLE_LAST_MAJOR_OPTIMIZING_INTERVAL =
+      defineGauge("table_last_major_optimizing_duration_mills")
+          .withDescription("Duration in milliseconds after last major optimizing")
+          .withTags("catalog", "database", "table")
+          .build();
+
+  public static final MetricDefine TABLE_LAST_FULL_OPTIMIZING_DURATION =
+      defineGauge("table_last_full_optimizing_duration_mills")
+          .withDescription("Duration in milliseconds after last full optimizing")
+          .withTags("catalog", "database", "table")
+          .build();
+
+  public static final MetricDefine TABLE_LAST_OPTIMIZING_DURATION =
+      defineGauge("table_last_optimizing_duration_mills")
+          .withDescription("Duration in milliseconds after last optimizing")
+          .withTags("catalog", "database", "table")
+          .build();
+
+  public static final MetricDefine TABLE_OPTIMIZING_REFRESHED_SNAPSHOT_SLIDE_INTERVAL =
```

Comment:
Is `table_optimizing_lag_duration_mills` a proper name?
And we should calculate it with: latestSnapshot.time - optimizedSnapshot.time.